scripts/docsgen/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/coder/envbuilder"
+)
+
+func main() {
+	readmePath := "README.md"
+	readmeFile, err := os.ReadFile(readmePath)
+	if err != nil {
+		panic("Error reading " + readmePath + " file")
+	}
+	readmeContent := string(readmeFile)
+	startIndex := strings.Index(readmeContent, "<!--- start-flags --->")
+	endIndex := strings.Index(readmeContent, "<!--- end-flags --->")
+	if startIndex == -1 || endIndex == -1 {
+		panic("Start or end marks not found in the file.")
+	}
+
+	options := envbuilder.Options{}
```

Comment:
This is the Go style for zero-value instantiations.

```suggestion
	var options envbuilder.Options
```